SYCL/OnlineCompiler/online_compiler_OpenCL.cpp

```diff
@@ -0,0 +1,41 @@
+// REQUIRES: opencl
+
+// RUN: %clangxx -fsycl -fsycl-targets=%sycl_triple %s -DRUN_KERNELS -lOpenCL -o %t.out
+// RUN: %CPU_RUN_PLACEHOLDER %t.out
+// RUN: %GPU_RUN_PLACEHOLDER %t.out
+// RUN: %clangxx -fsycl -fsycl-targets=%sycl_triple %s -lOpenCL -o %th.out
+// RUN: %RUN_ON_HOST %th.out
```

Comment:
```suggestion
// RUN: %HOST_RUN_PLACEHOLDER %th.out
```

Reply:
Fixed in a857f2c3e3ab1692e991dba0a5be5bb4ae753181